stl/inc/xlocnum

```diff
@@ -1019,10 +1019,6 @@ private:
                 _Seendigit = true;
             }
 
-            if (_Seendigit) {
-                *_Ptr++ = '0'; // put one back
-            }
-

```

Comment:
This branch is useless, as the integral and fraction part has been finished, and the exponent part will be written here:
https://github.com/microsoft/STL/blob/60f18856c56389001c38a9929ef37bd5c01c4e47/stl/inc/xlocnum#L1092-L1096